bin/check-mail-delay.rb

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env ruby
+#
+# Check for mail delays in the Postfix mail queue. This is split out into its
+# own check because it requires storing details about every message in the
+# queue in memory, which may not be desirable on heavily-trafficked systems.
+
+# ===
+#
+# Copyright (c) 2016, Jonathan Hartman <j@hartman.io>
+#
+# Released under the same terms as Sensu (the MIT license); see LICENSE
+# for details.
+#
+
```

Comment:
Can you use the standard header from http://sensu-plugins.io/docs/developer_guidelines.html